documentation/modules/ROOT/pages/transformations/outbox-event-router.adoc

```diff
@@ -182,12 +182,13 @@ transforms.outbox.type=io.debezium.transforms.outbox.EventRouter
 == Options for applying the transformation selectively
 
 In addition to the change event messages that a {prodname} connector emits when a database change occurs, the connector also emits other types of messages, including heartbeat messages, and metadata messages about schema changes and transactions.
-Because the structure of these other messages different from the structure of the change event messages that the SMT is designed to process, it's best to configure the connector to selectively apply the SMT, so that it processes only the intended data change messages.
+Because the structure of these other messages is different from the structure of the change event messages that the SMT is designed to process, it's best to configure the connector to selectively apply the SMT, so that it processes only the intended data change messages.
```

Comment:
Other SMTs contain different wording
```suggestion
Because the structure of these other messages differs from the structure of the change event messages that the SMT is designed to process, it's best to configure the connector to selectively apply the SMT, so that it processes only the intended data change messages.
```